### PR TITLE
fix(index): remove catch statement to allow rejection, always return promise

### PR DESCRIPTION
**Pull request by wzr1337**
Fixes https://github.com/dyaa/node-ssl-checker/issues/9

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -16,17 +16,16 @@ module.exports = (host, method, port) => {
 		host: host,
 		method: method || 'HEAD',
 		port: port || 443,
-		rejectUnauthorized: false,
-		agent: false,
```
Comment (dyaa) [inline index.js]: This was a fix, please check #7

Comment (wzr1337) [reply index.js]: This was not intended... I might have branched from the wrong branch (master)... I can fix it back in... It is just the single agent line isn't it?

WE should add a test case 😜

**Comment by wzr1337** [comment]
finally reverted  the un-intentional change ;)

**Comment by dyaa** [comment]
Thanks @wzr1337 :+1:

**Pull request merged**